product_docs/docs/pgd/5/security/pgd-predefined-roles.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: PGD predefined roles
+---
+
+PGD predefined roles are created when the BDR extension is installed. After BDR
+extension is dropped from a database, the roles continue to exist. You need to
+drop them manually if dropping is required. This practice allows PGD to be used
+in multiple databases on the same PostgreSQL instance without problem.
+
+The `GRANT ROLE` DDL statement doesn't participate in PGD replication. Thus,
+execute this on each node of a cluster.
```

Comment:
Should this be an admonition? Or can this note be removed from here? Do I have to run this command to create these roles? This needs more context.

Reply:
Not an admonition as this is literally the multiple database issue.

```suggestion
A `GRANT ROLE` DDL statement issued for a non replicated database doesn't participate in PGD replication. If the user/role is needed on other nodes, execute the command on each node of a cluster.
```